extensions/html-language-features/server/src/test/embedded.test.ts

```diff
@@ -128,4 +128,11 @@ suite('HTML Embedded Support', () => {
 		assertEmbeddedLanguageContent('<div onKeyUp=return\n/><script>foo();</script>', 'javascript', '             return;\n          foo();         ');
 	});
 
+	test('Script content - HTML escape characters', function (): any {
```

Comment:
What happens if the input does not contain any escape sequences? It might be worth adding a test to ensure the input is returned unmodified in that case.

Reply:
Thanks for the feedback! I will add another test to confirm this expectation.